src/components/chips/js/chipsDirective.js

```diff
@@ -199,7 +199,7 @@
               role="option"\
               aria-selected="{{$mdChipsCtrl.selectedChip == $index}}" \
               aria-posinset="{{$index}}"\
-              ng-click="!$mdChipsCtrl.readonly && $mdChipsCtrl.focusChip($index)"\
+              ng-click="!$mdChipsCtrl.readonly && $mdChipsCtrl.focusChip($index, $event)"\
```

Comment:
I'm kind of worried about stopping propagation here. Can't you below in the other `click` event just check if a chip or the input is selected?

Right now, it seems like it also resets the selected item on every input element click.

Reply:
Clearing the selected chip on input focus or md-chips blur seems to be the expected behavior (as it's in 1.1.1, 1.1.9, and HEAD). This is also consistent with Angular Material.

Reply:
Good tip on the check in the directive's click handler. That works well!